releng/raw-binaries/Dockerfile

```diff
@@ -43,11 +49,19 @@ RUN mkdir -p /opt/osxcross && \
 
 ENV PATH="/opt/osxcross/target/bin:${PATH}"
 
-# Wrapper to pull a version of pkg-config at runtime
-ENV PKG_CONFIG=/usr/local/bin/pkg-config.sh
+RUN cd /tmp && \
+    git clone https://github.com/richfelker/musl-cross-make.git && \
+    cd musl-cross-make && \
+    git checkout v0.9.9 && \
+    make MUSL_VER=1.1.24 TARGET=aarch64-unknown-linux-musl DL_CMD="curl -C - -L -o" install && \
+    mv output /opt/musl-cross && \
+    rm -rf /tmp/musl-cross-make
 
-# install go-junit-report for unit tests
-RUN GO111MODULE=on go get github.com/jstemmer/go-junit-report@v0.9.1
+ENV PATH="/opt/musl-cross/bin:$PATH"
+
+# Make sure we have the newer version of pkg-config that works with windows,
+# and install go-junit-report for unit tests
+RUN GO111MODULE=on go get github.com/jstemmer/go-junit-report@v0.9.1 github.com/influxdata/pkg-config@v0.2.7
```

Comment:
I have similar questions about the go-junit-report project. Also, it looks like it hasn't been touched in a while, which makes me think it isn't actively maintained. It might be worth putting a comment in here about what we use this for and why, in the event one of our changes introduces compatibility problems.

Reply:
We use this to transform the go-test output into a junit-style xml report suitable for ingest by the Jenkins `junit` command, which tracks which tests (if any) are failing during each build.

Reply:
For future work: 2.x uses `gotestsum` instead of `go test` to generate JUnit reports